test/unit/test_unit_tests.py

```diff
@@ -418,8 +424,62 @@ def test_exclude_classes_covered_by_dev_test(self) -> None:
 
             config['dev_tests']['coverage_threshold'] = 96
             generate.exclude_classes_covered_by_dev_test(config, dir_util.get_app_output_dir(app_name))
-            self.assertTrue(config['generate']['excluded_class_list'] == test_apps[app_name]['covered_classes'])
+            self.assertTrue(set(config['generate']['excluded_class_list']) == set(test_apps[app_name]['covered_classes']))
+
+        self.__assert_no_artifact_at_cli(test_apps.keys())
+
+    def test_augment_on_empty_test_suite(self) -> None:
+        """Test that we can augment on an empty ctd test suite"""
+        test_apps = self.bad_path_test_apps
 
+        for app_name in test_apps.keys():
```

Comment:
it is redundant to test the empty test suite case on all apps. One app is enough.

Reply:
not sure I understand the comment, only one app is tested:
        **test_apps = self.bad_path_test_apps
        for app_name in test_apps.keys():
            dir_util.cd_cli_dir()**

Reply:
Right, missed that. But better to get the app directly by its name and not iterate over bad paths test apps, because if we add more bad path apps to this test file, we don't want them to be automatically added to this test method. We need just a single corner case.